src/main/kotlin/lotto/LottoMachine.kt

```diff
@@ -0,0 +1,13 @@
+package lotto
+
+class LottoMachine(private val lottoMaker: LottoMaker) {
+
+    fun buyLotto(count: Int): Lottos {
+        val lottos = mutableListOf<Lotto>()
+        repeat(count) {
+            lottos.add(lottoMaker.make())
+        }
+
+        return Lottos(lottos)
```

Comment:
```suggestion
        return Lottos(lottos.toList())
```
외부에 전달할 때에는 원본이 손상되지 않도록 방어적 복사를 활용해보면 어떨까요?

Reply:
해당 객체가 상태값으로 가지고 있지 않고 있고 생성해서 넘겨주고 있기 때문에 별도의 방어적 복사는 필요 없을 거라 생각을 했는데요~
이런 경우 (생성만 하는 객체) 에도 방어적 복사를 하는 것이 좋을까요?